client_operations.go

```diff
@@ -341,27 +341,29 @@ func (list *ListSitesParams) URLParams() []KeyValue {
 // API Documentation: https://developers.recurly.com/api/v2019-10-10#operation/list_sites
 //
 // Returns: A list of sites.
-func (c *Client) ListSites(params *ListSitesParams) *SiteList {
+func (c *Client) ListSites(params *ListSitesParams, opts ...Option) *SiteList {
 	path, err := c.InterpolatePath("/sites")
 	if err != nil {
 		// NOOP in 3.x client
 	}
+	requestOptions := NewRequestOptions(opts...)
```

Comment:
as discussed, we want to first apply the params in *listSitesParams, then apply the opts on top